server/logger.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package server
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var logger *log.Logger
+
+func init() {
+	logger = log.New()
+	logger.SetFormatter(&log.TextFormatter{
+		FullTimestamp: true,
+	})
+	// Output to stdout instead of the default stderr.
+	log.SetOutput(os.Stdout)
+}
+
+type logrusWriter struct {
```

Comment:
Do we need this? I don't see any usage.

Reply:
Yep, good catch. That was a copy-pasta that we don't seem to need.